crates/astria-composer/src/config.rs

```diff
@@ -15,22 +12,10 @@ use serde::{
     Serializer,
 };
 
-/// Utility function to read the application's config in one go.
-///
-/// This includes parsing the command line args, getting all environment variables.
-/// This function will short circuit binary execution when `--help` or `--version`
-/// is provided, or if the command line arguments could not be read.
-///
-/// # Errors
-///
-/// An error is returned if the config could not be read.
-pub fn get() -> Result<Config, figment::Error> {
-    Config::from_environment()
-}
-
 /// The high-level config for creating an astria-composer service.
-#[derive(Debug, Deserialize, Serialize)]
+#[derive(Serialize, Deserialize)]
 #[serde(deny_unknown_fields)]
```

Comment:
The tests in `astria-utils` passing rely on `deny_unknown_fields` being present, right?

It's pretty surprising that that's the case (and honestly, I should have added that `deny_unknown_fields` in the first place). Is there a way to not rely on that?

Reply:
Yeah, it's just that the initial implementation was not good. We (== I) should not have enforced this at runtime, only at test-time.